persistent/Database/Persist/Quasi/Internal.hs

```diff
@@ -797,6 +620,12 @@ mkUnboundEntityDef ps parsedEntDef =
     idSqlType =
         maybe SqlInt64 (const $ SqlOther "Primary Key") primaryComposite
 
+commentedField
+    :: PersistSettings -> ([Token], Maybe Text) -> Maybe UnboundFieldDef
+commentedField ps (tokens, mCommentText) = do
+    unb <- takeColsEx ps (tokenContent <$> tokens)
+    pure $ unb{unboundFieldComments = mCommentText}
```

Comment:
The interface of this function is tightly coupled to its one call-site, which suggests it may not be a great top-level definition. Locating it a bit more closely in the `where` or a `let` as a helper function may be more appropriate.